### ISSUE TITLE
Chart titles truncated but only when running under Linux

### ISSUE THREAD

**Issue by FusionSnowball** [EPPlusSoftware/EPPlus#1148]
I've encountered a bug that only happens when running on a Linux host (5.10.102.1-microsoft-standard-WSL2). It behaves as expected under Windows. When I set a chart's title using ExcelChartTitle.Text the last character of the title is removed. This effects the main chart title and axis titles. 

As a workaround I can add a space to end of the title, which is then removed by the bug, when the problem is detected.

I've seen this using EPPlus 6.2.11 and 7.0.0.

Example code (includes workaround)... 

```
private static void SetChartTitle(ExcelChartTitle targetTitle, string titleValue)
{
     targetTitle.Text = titleValue;

    if (targetTitle.Text != titleValue)
    {
        targetTitle.Text = titleValue + " "; // space then removed by bug leaving the title we want
    }
}
```

Any ideas?

**Comment by swmal** [comment]
I don't see any obvious reason to why this would be different depending on the underlying o/s. 

What type of chart are you creating when this happens (you are probably calling` worksheet.Drawings.Add[chart type]Chart(...)` at some point)? There are different classes that handles the Text property depending of the type of chart.

**Comment by FusionSnowball** [comment]
It is created via worksheet.Drawings.AddChart where the eChartType parameter is Line, AreaStacked, BarStacked, Area, or LineStacked (all have the same problem).

**Comment by FusionSnowball** [comment]
I've discovered some more information on this problem. I'm seeing the error as part of a unit test that creates the Excel file then reads it back in to check it has worked. If I intercept the Excel file in the middle the titles are complete. It's when it reads the file back in that it's getting truncated only on Linux. 

It's doing this to read the file...
```
var _book =new ExcelPackage(memoryStream)
var sheet = _book.Workbook.Worksheets["Report"];
var charts = sheet.Drawings.Where(d => d is ExcelChart).Cast<ExcelChart>().ToList();
var chart = charts.ElementAt(0);
var Title = chart.Title.Text; // this is missing the last character
```

When I check the title text at the time it's set (the if statement in the first post here) it's also truncated. A problem with the property getter?

**Comment by swmal** [comment]
I think we have found the cause of this:

https://github.com/EPPlusSoftware/EPPlus/blob/develop7/src/EPPlus/Style/ExcelParagraphCollection.cs#L205

We should use `Environment.NewLine` here instead of having a hardcoded removal of the last two characters. A fix is on the way.

**Comment by swmal** [comment]
Once PR's #1154 and #1155 has been merged into the develop branches you can test the fix via our [develop nuget feed](https://github.com/EPPlusSoftware/EPPlus/wiki/Using-our-develop-Nuget-feed).

**Comment by FusionSnowball** [comment]
I can confirm that this change has fixed the problem, thanks. When will it appear in a stable release on nuget.org?

**Comment by swmal** [comment]
This was released in v6.2.13 and v7.0.2, see https://epplussoftware.com/en/Developers/MinorFeaturesAndIssues
